sale_line_refund_to_invoice_qty/wizards/account_move_reversal.py

```diff
@@ -8,7 +8,7 @@ class AccountMoveReversal(models.TransientModel):
 
     sale_qty_to_reinvoice = fields.Boolean(
         string="This credit note will be reinvoiced",
-        default="True",
+        default=lambda self: self.env.company.reinvoice_credit_note_default,
```

Comment:
the account.move.reversal wizard already has a company field. What you could do is to put the default of the sale_qty_to_reinvoice in the `def default_get(self, fields)` method of the wizard, where the company is filled. Check: https://github.com/odoo/odoo/blob/14.0/addons/account/wizard/account_move_reversal.py#L39

Reply:
@JordiMForgeFlow I updated with your suggestion. Let me know what you think about my implementation.